ci/travis/test-worker-in-container.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+export STORAGE_DRIVER=vfs
+podman load --input /var/lib/containers/images.tar
+
+cat > /ray/docker/ray-nest-container/test-Dockerfile << EOF
+FROM rayproject/ray-nest-container:nightly-py36-cpu
+RUN $HOME/anaconda3/bin/pip --no-cache-dir install pandas
+EOF
+
+podman build -f /ray/docker/ray-nest-container/test-Dockerfile -t rayproject/ray-nest-container:nightly-py36-cpu-pandas .
+
+export RAY_BACKEND_LOG_LEVEL=debug
+"$HOME/anaconda3/bin/pip" install --no-cache-dir pytest
+pytest /ray/python/ray/tests/test_actor_in_container.py -s
```

Comment:
please use bazel to launch test instead of pytest directly

Reply:
this will enable better reporting. please also add `    - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/travis/upload_build_info.sh; fi }; trap cleanup EXIT` to the steps so we are uploading correct logs.